sceptre/template.py

```diff
@@ -261,3 +263,30 @@ def _create_bucket(self, region, bucket_name, connection_manager):
                     }
                 }
             )
+
+    @staticmethod
+    def _render_jinja_template(template_dir, filename, jinja_vars):
```

Comment:
So I considered this, but I think we need the `jinja` in there to correctly namespace it against the `Template` class. `_render_template` sounds like it should render the CFN template, whereas we're actually rendering a Jinja template